lightning/src/ln/channel.rs

```diff
@@ -61,6 +61,12 @@ pub struct ChannelValueStat {
 	pub counterparty_dust_limit_msat: u64,
 }
 
+#[derive(Clone, Copy, PartialEq)]
+enum InboundFeeUpdateState {
+	RemoteAnnounced,
+	AwaitingRemoteRevokeToAnnounce,
+}
```

Comment:
As is, it's a bit confusing to have `InboundFeeUpdateState` be set here for outbound channels. Could we change this to something like:

```
enum FeeUpdateState {
    RemoteAnnounced,
    AwaitingRemoteRAA,
    OutboundChannel,
}
```

Reply:
Hmm, its always ignored for outbound channels, no? I guess we could set it to an `Option`, which feels a bit cleaner than a new enum variant, I think, but its all a little awkward.